src/core/binding.c

```diff
@@ -1715,6 +1715,24 @@ QuicBindingUnreachable(
     }
 }
 
+_IRQL_requires_max_(DISPATCH_LEVEL)
+_Function_class_(CXPLAT_DATAPATH_ROUTE_RESOLUTION_CALLBACK)
+void
+QuicBindingRoute(
```

Comment:
If the context is the connection, I think it's fine we eliminate `QuicBindingRoute` entirely and just pass `QuicConnQueueRouteCompletion` as the callback pointer.